packages/apple-pay-integration/e2e/ApplePaySessionCustomerStepMockObject.ts

```diff
@@ -0,0 +1,165 @@
+const addApplePaySessionToChrome = () => {
+    class ApplePaySessionCustomerStep implements ApplePaySession {
+        version: number;
+        paymentRequest: ApplePayJS.ApplePayPaymentRequest;
+
+        static STATUS_SUCCESS = 1;
+        static STATUS_FAILURE = 2;
+
+        static supportsVersion(_versionNumber: unknown) {
+            console.log('supportsVersion', _versionNumber)
+            return true;
+        }
+
+        constructor(version, paymentRequest) {
+            this.version = version;
+            this.paymentRequest = paymentRequest;
+        }
+
+        addEventListener(_type: string, _callback: EventListenerOrEventListenerObject | null, _options?: boolean | AddEventListenerOptions | undefined): void {
```

Comment:
Don't prefix the parameter with an underscore if you're actually using it.

Reply:
Removed the prefix